install/parse-cli.sh

```diff
@@ -17,7 +17,11 @@ Options:
  --skip-commit-check    Skip the check for the latest commit when on the master
                           branch of a \`self-hosted\` Git working copy.
  --skip-user-creation   Skip the initial user creation prompt (ideal for non-
-                          interactive installs).
+                          	interactive installs).
+ --skip-sse42-requirements
+			Skip checking that your environment meets the
+			  requirements to run Sentry. Only do this if you know
```

Comment:
```suggestion
			  requirements to run Clickhouse on x86. Only do this if you know
```
The real need of this is clickhouse uses SSE 4.2 on x86 (not on ARM)